pipelines/crawler/rf_cnpj/utils.py

```
@@ -0,0 +1,1030 @@
+        column (str): The name of the column to retrieve unique keys from.
+    Returns:
+        pd.DataFrame: A DataFrame containing the unique keys from the specified table and column.
+    """
+    query = f"""WITH tmp_split AS(
+    SELECT
+        split(safe_cast({column} AS STRING),",") AS chave 
+    FROM `basedosdados-dev.br_rf_cnpj_staging.{table_id}`
+    )
+    SELECT DISTINCT chave
+    FROM tmp_split,
+    UNNEST(chave) AS chave"""
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)